docs/ReleaseNotes.md

```diff
@@ -18,6 +18,19 @@ The included licenses apply to the following files:
 
 ## Changelog
 
+### Version 1.8.2407
+
+This cumulative release contains numerous bug fixes and stability improvments.
+
+Here are some highlights:
+
+- Retain alignment when lowering groupshared matrices by @pow2clk in [#6589](https://github.com/microsoft/DirectXShaderCompiler/pull/6589)
+- [Optimization] More aggressive reassociations by @lizhengxing in [#6626](https://github.com/microsoft/DirectXShaderCompiler/pull/6626)
+- Fix RemoveUnstructuredLoopExits when an exiting edge jumps out multiple levels of loops by @dneto0 in [#6668](https://github.com/microsoft/DirectXShaderCompiler/pull/6668)
+- [SPIR-V] Implement WaveMutliPrefix* by @sudonatalie in [#6608](https://github.com/microsoft/DirectXShaderCompiler/pull/6608)
+- [SPIR-V] Add support for SampleCmpLevel by @Keenuts in [#6618](https://github.com/microsoft/DirectXShaderCompiler/pull/6618)
+- Avoid adding types to default namespace by @s-perron in [#6700](https://github.com/microsoft/DirectXShaderCompiler/pull/6700)
```

Comment:
Rather than linking pull requests and specifically calling out usernames, can we list the issues?

Calling out usernames in our release notes ends up broadcasting GitHub usernames around which is not great, and for end users seeing the context of the issue that is resolved is probably more valuable than the change which resolved it.

Reply:
Updated.